web-ui/src/app/pipes/calendar-date.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DatePipe } from '@angular/common'
+
+@Pipe({
+  name: 'calendarDate'
```

Comment:
I would give a specific name like `explorerDateTime`, the pipe renders the date and time, for me `calendarDate` is a very generic name while the pipe is specific for our application usage.